docs/source/joining_datasets.md

```diff
@@ -18,7 +18,7 @@ splits:
                 join:
                     - path: ds1
                     - path: ds2
-                join_type:
+                joiner:
                     __module__: megatron.energon
                     __class__: CaptioningSample
```

Comment:
I believe this is still wrong. Should be the joiner method, no?

Reply:
See below